tests/test_base.py

```diff
@@ -56,10 +56,21 @@ def teardown_method(self, method):
     def _ensure_queues(
         self, queued=None, active=None, error=None, scheduled=None
     ):
+
```

Comment:
These lines aren't related to the other functionality in this PR, but the improved error message helped me a bit while debugging tests.